srcpkgs/n/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'n'
 pkgname=n
-version=9.2.0
+version=9.2.1
 revision=1
 build_style=gnu-makefile
 depends="curl"
```

Comment:
This is a simple `bash` script, that uses `curl` or `wget` to fetch `glibc`-only nodejs binaries distribution, then uses `tar` and `gzip` or `xz` to unpack it out.

So:
- **Personally**, I will not merge this update
- Its dependency needs to be updated
- It should be limited to glibc only.

Reply:
I think the most recent revision should cover everything